checks/core.py

```diff
@@ -136,3 +136,16 @@ def checkSafeMode(lines):
     if search('Safe Mode enabled.', lines):
         return [LEVEL_WARNING, "Safe Mode Enabled",
                 """You are running OBS in Safe Mode. Safe Mode disables third-party plugins and prevents scripts from running."""]
+
+
+def checkSnapPackage(lines):
+    isDistroNix = search('Distribution:', lines)
+
+    if len(isDistroNix) <= 0:
+        return
+
+    distro = isDistroNix[0].split()
+    # Snap Package logs "Ubuntu Core" as distro, so it gets split halfway
+    if distro[2] == '"Ubuntu' and distro[3] == 'Core"':
+        return [LEVEL_CRITICAL, "Snap Package",
+                "You are using the Snap Package. This is an unofficial package not maintained by the OBS Team and is shipped with a variety of (often conflicting) Third-Party plugins. OBS support is unable to assist with issues arising out of the usage of this package and therefore recommends following our <a href=\"https://obsproject.com/download#linux\">Install Instructions</a>."]
```

Comment:
Change this message to the notice and link from [the snapcraft page](https://snapcraft.io/obs-studio), or adapt their wording:
> NOTE: This is a community-supported modified build of OBS Studio; please file issues on the Snapcrafters GitHub: https://github.com/snapcrafters/obs-studio/issues

Reply:
If this change is made to the verbiage, we can merge after.

Reply:
Should I still mention the official install instructions?

Reply:
I'd prefer if it was dropped to warning, not critical. And yes, you can leave the part about us recommending the official install methods, with a slight change to the wording here:

`OBS may be unable to assist with issues arising out of the usage of this package and therefore recommends following our <a href=\"https://obsproject.com/download#linux\">Install Instructions</a>."`

Reply:
Just following up on @Fenrirthviti 's ask, the current wording is now:
```
OBS support may be unable to assist with issues arising out of the usage of this package and therefore recommends following our <a href=\"https://obsproject.com/download#linux\">Install Instructions</a>.
```
compared to the ask:
```
OBS may be unable to assist with issues arising out of the usage of this package and therefore recommends following our <a href=\"https://obsproject.com/download#linux\">Install Instructions</a>."
```
It looks like the additional word "support" was carried over from the original wording on this PR.

@Fenrirthviti if that's okay, then I'm okay with these changes.

Reply:
Yes, I think dropping "support" here is correct just to keep consistent with other verbiage.

Reply:
@mihawk90 Please drop the word "support" here, as asked above.

Reply:
Fixed.